moshi/moshi/server.py

```diff
@@ -223,27 +236,53 @@ def main():
     app = web.Application()
     app.router.add_get("/api/chat", state.handle_chat)
     static_path: None | str = None
+
     if args.static is None:
         log("info", "retrieving the static content")
         dist_tgz = hf_hub_download("kyutai/moshi-artifacts", "dist.tgz")
         dist_tgz = Path(dist_tgz)
         dist = dist_tgz.parent / "dist"
         if not dist.exists():
             with tarfile.open(dist_tgz, "r:gz") as tar:
-                tar.extractall(path=dist_tgz.parent)
+                safe_extract(tar, path=dist_tgz.parent)
         static_path = str(dist)
-    elif args.static != "none":
+    elif args.static.lower() != "none":
         # When set to the "none" string, we don't serve any static content.
         static_path = args.static
+
     if static_path is not None:
-        async def handle_root(_):
-            return web.FileResponse(os.path.join(static_path, "index.html"))
+        # Resolve the static path to an absolute path
+        static_path = Path(static_path).resolve()
```

Comment:
I think the typer will complain with this. I see you didn't run pre-commit hooks

Reply:
[![precommit](https://github.com/kyutai-labs/moshi/actions/workflows/precommit.yml/badge.svg)](https://github.com/kyutai-labs/moshi/actions/workflows/precommit.yml)

Reply:
[![precommit](https://github.com/kyutai-labs/moshi/actions/workflows/precommit.yml/badge.svg)](https://github.com/kyutai-labs/moshi/actions/workflows/precommit.yml)